src/typings/utilities.ts

```diff
@@ -0,0 +1,5 @@
+// If all properties are optional, Make at least one required
+export type RequireAtLeastOne<T> = {
```

Comment:
Can't see this type used in the PR. Was it added automatically?

Reply:
It was a remnant of my original commit. I've deleted it.